internal/mode/static/nginx/runtime/runtime_manager_test.go

```diff
@@ -0,0 +1,95 @@
+package runtime_test
+
+import (
+	"github.com/go-logr/logr"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime"
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/nginx/runtime/runtimefakes"
+	ngxclient "github.com/nginxinc/nginx-plus-go-client/client"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("NGINX Runtime Manager", Ordered, func() {
+	var (
+		manager         runtime.Manager
+		upstreamServers []ngxclient.UpstreamServer
+		upstreamServer  ngxclient.UpstreamServer
+		logger          logr.Logger
+		ngxPlusClient   *runtimefakes.FakeNginxPlusClient
+	)
+
+	var (
+		MaxConnsMock = 150
+		MaxFailsMock = 20
+		WeightMock   = 10
+		BackupMock   = false
+		DownMock     = false
+	)
+
+	BeforeAll(func() {
+		upstreamServer = ngxclient.UpstreamServer{
+			ID:          1,
+			Server:      "10.0.0.1:80",
+			MaxConns:    &MaxConnsMock,
+			MaxFails:    &MaxFailsMock,
+			FailTimeout: "test",
+			SlowStart:   "test",
+			Route:       "test",
+			Backup:      &BackupMock,
+			Down:        &DownMock,
+			Drain:       false,
+			Weight:      &WeightMock,
+			Service:     "test",
+		}
+
+		upstreamServers = []ngxclient.UpstreamServer{
+			upstreamServer,
+		}
+
+		logger = logr.New(GinkgoLogr.GetSink())
```

Comment:
This can just be initialized as `zap.New()`, like in other tests. Using the `controller-runtime` zap library.

Reply:
make sense, will update, 
do you want me to add an update also here: https://github.com/nginxinc/nginx-gateway-fabric/blob/main/internal/mode/static/nginx/runtime/manager_test.go#L18 
I went by this approach here

Reply:
Ah, yes, if possible please make that change.

Reply:
will do